projects/plugins/jetpack/modules/masterbar/admin-menu/upsell-nudge.js

```diff
@@ -0,0 +1,85 @@
+( () => {
+	function requestJitm() {
+		if ( window.wpApiSettings.root.indexOf( 'https://public-api.wordpress.com/' ) === 0 ) {
```

Comment:
This function is polymorphic and performs requests in 2 different ways:

* To the remote Jetpack jitm endpoint if ran from a WP.com simple site.
* To the built-in Jetpack jitm endpoint if ran from a Jetpack / Atomic site

Reply:
Not sure of current browser support and transpilation/polyfill settings on Jetpack, but `startsWith` would be nice here, if available.

Reply:
Could be worth including a simple comment similar to your above Github comment in the code: `// Use remote Jetpack JITM endpoint on WP.com Simple sites.`

Reply:
> Not sure of current browser support and transpilation/polyfill settings on Jetpack, but startsWith would be nice here, if available.

I intentionally didn't use it, because [docs say](https://github.com/Automattic/jetpack/blob/master/docs/coding-guidelines.md#versions-supported):

> We support the latest two versions of all major browsers, except IE, where we currently only support 11 and Edge.

and AFAIK IE11 doesn't support `String.prototype.startsWith()`. 

> Could be worth including a simple comment similar to your above Github comment in the code:

Added in the latest commit.